roles/openshift_logging_kibana/tasks/main.yaml

```diff
@@ -246,6 +261,30 @@
     - "{{ tempdir }}/templates/kibana-dc.yaml"
     delete_after: true
 
+- name: Generate Memcached DC template
+  template:
+    src: memcached.j2
+    dest: "{{ tempdir }}/templates/memcached-dc.yaml"
+  vars:
+    component: "{{ kibana_component }}"
+    logging_component: kibana
+    deploy_name: "memcached-{{ kibana_name }}"
+    image: "{{ openshift_logging_image_prefix }}logging-memcached:{{ openshift_logging_image_version }}"
```

Comment:
There will need to be some logic in defaults and such to allow the image version to be pegged at something specific to avoid unexpected behavior when pods restart and new images are available